.github/actions/frontend-shared-version/index.js

```diff
@@ -0,0 +1,58 @@
+'use strict'
+
+const core = require('@actions/core');
+const util = require('util');
+
+const exec = util.promisify(require('child_process').exec);
+
+/**
+ * Returns true is there is a non-zero length diff between the last published version
+ * and master within the frontend-shared/ folder.
+ */
+async function haveChangesOccurred() {
+  const parentPackage = core.getInput('parent-package');
+  const versionResult = await exec(`npm view ${parentPackage} version`);
```

Comment:
The package tag to look at isn't specified here. According to https://docs.npmjs.com/cli/v6/commands/npm-view it defaults to `latest`, which is what we want.

Reply:
I'll add @latest so its clear